bin/container-test

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
```

Comment:
Whilst error on exit catches a number of issues, it doesn't cover undefined variables or the pipe return code gotcha. As such I tend to always use `set -euo pipefail` - see:
http://redsymbol.net/articles/unofficial-bash-strict-mode/

Reply:
`pipefail` doesn't exist on sh, only on bash. And circle doesn't make bash available in their default images.

Reply:
> And circle doesn't make bash available in their default images.

Ah TIL